aviary/models/test_aircraft/aircraft_for_bench_GwGm_solved.csv

```diff
@@ -151,7 +151,7 @@ mission:summary:fuel_flow_scaler,1,unitless
 mission:takeoff:decision_speed_increment,10,kn
 mission:takeoff:rotation_speed_increment,5,kn
 mission:taxi:duration,0.1677,h
-settings:equations_of_motion,solved
```

Comment:
I believe this one is supposed to be "solved", @johnjasa can you confirm?

Reply:
The reason I changed it was I got this error:

![image](https://github.com/OpenMDAO/Aviary/assets/165826178/f3bd552d-69ac-4b3b-9cd6-acebc2af187b)

When I checked enums.py I only saw 3 options and so changed 'solved' to 'solved_2dof' as I guessed they were the same. 
![image](https://github.com/OpenMDAO/Aviary/assets/165826178/346bffd0-731e-40d0-be5a-3203c3310220)

Reply:
Yes, `solved` has been changed to `solved_2dof`. That `aircraft_for_bench_GwGm_solved.csv` should just be removed as it's defunct